.gitignore

```diff
@@ -53,5 +53,5 @@ node_modules
 .pnp.*
 
 ### Auto-generated ###
-src/main/resources/icons
```

Comment:
Check commit description. IntelliJ doesn't properly does resource isolation, so if the icons are in the same subfolder of `resources`. it can't be guaranteed that IntelliJ won't just randomly pick an icon from another plugin with the same icons location.